data_structures/LinkedList/Circular Linked List

```diff
@@ -0,0 +1,109 @@
+class Node:
+    def __init__(self, data):
+       self.data = data
+       self.next = None
+       self.prev = None
+ 
+ 
+class CircularDoublyLinkedList:
+    def __init__(self):
+        self.first = None
+ 
+    def get_node(self, index):
+        current = self.first
```

Comment:
```python
if sefl.first is None :
    return None
```

add this to avoid errors on invalid inputs like **insert _&lt;data&gt;_ after _&lt;index&gt;_** on **empty list**